users/stdiscosrv.rst

```diff
@@ -0,0 +1,69 @@
+Synthing Discovery Server
+=========================
+
+Synopsis
+--------
+
+::
+
+    stdiscosrv [-cert=<file>] [-db-backend=<string>] [-db-dsn=<string>] [-debug] [-http] [-key=<string>]
+               [-limit-avg=<int>] [-limit-burst=<int>] [-limit-cache=<int>] [-listen=<address>]
+               [-stats-file=<file>]
+
+Description
+-----------
+
+Syncthing relies on a discovery server to find peers. Anyone can run a discovery
```

Comment:
I am sure this is not the best description. You might be better than me to explain this (and not lie! which I might by accident.)